app/editor/boot.js

```diff
@@ -792,6 +792,17 @@ thin.init_ = function() {
       }
     });
 
+    // Mouse wheel zoom
+    document.body.addEventListener('mousewheel', function(e){
```

Comment:
Although Ctrl+mouse wheel zoom events can also occur in the menu bar and on the right pane, I think it's more natural to trigger them only on the canvas. Could you change it as follows?

```suggestion
    main.getMain().getElement().addEventListener('mousewheel', function(e) {
```